homeassistant/helpers/device_registry.py

```diff
@@ -527,6 +546,7 @@ def _data_to_save(self) -> Dict[str, List[Dict[str, Any]]]:
                 "area_id": entry.area_id,
                 "name_by_user": entry.name_by_user,
                 "disabled_by": entry.disabled_by,
+                "suggested_area": entry.suggested_area,
```

Comment:
Should we still save it ?

Reply:
```suggestion
```

Reply:
Was super useful for manual testing, but I don't think we need to keep it around.

Reply:
Will remove and push a test to verify it doesn't survive a save/reload cycle